### DISCUSSION TITLE
which part of the code is made for calculating and adding scores

### DISCUSSION THREAD

**Discussion by jeffreyyang39** [KadeArchive/Kade-Engine#2729] [Q&A]
Exact to which file(s) ,which class(es). Thanks a lot!

**Comment by glowsoony** [comment]
Adding scores if as like accuracy is Rating.hx and for "ratings like 'Sick'"
here is a list if it was for adding ratings like 'sick':
-playstate.hx
-ratings.hx
-convertscore.hx

for Ratings like accuracy
-Ratings.hx
